python_on_whales/components/image/cli_wrapper.py

```diff
@@ -202,6 +202,80 @@ def __init__(self, client_config: ClientConfig):
             client_config
         ).build
 
+    def legacy_build(
+        self,
+        context_path: ValidPath,
+        add_hosts: Dict[str, str] = {},
+        build_args: Dict[str, str] = {},
+        cache: bool = True,
+        file: Optional[ValidPath] = None,
+        labels: Dict[str, str] = {},
+        network: Optional[str] = None,
```

Comment:
I think maybe this should accept a list? Not sure

Reply:
The `docker image build --help` seems to disallow multiple inputs, so I believe multiple networks only works with buildx.